site2/website/contributing.md

```diff
@@ -15,11 +15,12 @@ We use a review-then-commit workflow in Pulsar for all contributions.
 
 **For all contributions:**
 
-1. **Code:** The best part ;-)
```

Comment:
For an open-source community, no one is the "best" part, every part (code, doc, evangelist, operation etc) is essential, they should be treated equally.